ACP2E-228: [Documentation] Add notice that the cookie path can't contain any cookie parameters except path to folders (#1535)
magento/merchdocs: src/configuration/general/web.md

|Field|[Scope]({% link configuration/scope.md %})|Description|
|--- |--- |--- |
|Cookie Lifetime|Store View|Determines how long a cookie can exist before it is automatically deleted; Default value is 3600 seconds (1 hour)|

> |Cookie Path|Store View|Specifies the folders on the server where Commerce cookies can be used. To make Commerce cookies available everywhere in the installation, set the Cookie Path to a single forward slash: `/`. It can't contain any other cookie parameters, except the cookie path.|

Reviewer: ```suggestion
|Cookie Path|Store View|Specifies the folders on the server where Commerce cookies can be used. To make Commerce cookies available everywhere in the installation, set the Cookie Path to a single forward slash: `/`. This value can contain only the cookie path, and **_cannot_** contain any other cookie parameters.|
```

Author: Requested changes were applied successfully

> |Cookie Domain|Store View|Determines if Commerce cookies are available to subdomains. For example, to support mysubdomain.domain.com, enter the name of your domain with a period at the beginning, like `.domain.com`. It can't contain any other cookie parameters, except the cookie domain.|

Reviewer: ```suggestion
|Cookie Domain|Store View|Determines if Commerce cookies are available to subdomains. For example, to support mysubdomain.domain.com, enter the name of your domain with a period at the beginning, like `.domain.com`. This value can contain only the cookie domain, and **_cannot_** contain any other cookie parameters.
```
